02-Python-1.ipynb

```diff
@@ -0,0 +1,844 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Python 1 - Introduction to Python\n",
+    "\n",
+    "<br/>\n",
+    "\n",
+    "<section class=\"objectives panel panel-warning\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-certificate\"></span> Learning Objectives </h2>\n",
+    "</div>\n",
+    "<ul>\n",
+    "<li> Get used to the IPython Notebook. </li>\n",
+    "<li> Understand what a library is and how to use it.</li>\n",
+    "<li> Read tabular data from a file into a program.</li>\n",
+    "<li> Assign values to variables.</li>\n",
+    "<li> Select individual values and subsections from data.</li>\n",
+    "<li> Perform operations on arrays of data.</li>\n",
+    "<li> Display simple graphs.</li>\n",
+    "</ul>\n",
+    "\n",
+    "</section>\n",
+    "\n",
+    "\n",
+    "\n",
+    "## IPython Notebook\n",
+    "\n",
+    "IPython notebook is a web based interactive computational environment with code, text, mathematics, plot and exectution all in the same place. Its how we'll be doing the whole course, though we wouldn't use it for more script based programming, for that we suggest you use Spyder or your pet text editor.\n",
+    "\n",
+    "A few shortcuts to make your life easier:\n",
+    "\n",
+    "- crtl + enter - Run the contents of this cell\n",
+    "- B - insert cell below this one when whole cell is selected\n",
+    "- esc - moves from this window to select the whole box\n",
+    "- enter - will move you back in\n",
+    "- M - changes the whole box to markdown\n",
+    "- Y - changes the whole box to code\n",
+    "\n",
+    "## Libraries\n",
+    "\n",
+    "Words are useful, but what’s more useful are the sentences and stories we build with them. Similarly, while a lot of powerful tools are built into languages like Python, even more live in the libraries they are used to build.\n",
+    "\n",
+    "Numpy is one of the essential libraries we use in python. We can import it as so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import numpy\n",
+    "from __future__ import division, print_function"
```

Comment:
@Cadair says the lines should be inverted, `__future__` should go in the first line.  It may break things otherwise...